### ISSUE TITLE
Support reading of multiple values at once.

### ISSUE THREAD

**Issue by TheChatty** [golles/ha-kamstrup_403#29]
### Did you read the instructions?

- [X] I have read the [README](../blob/master/README.md).

### The request

I simply tried to read multiple values and it worked. I documented it [here](https://tasmota.github.io/docs/Smart-Meter-Interface/#kamstrup-multical-4xx-6xx-8xx).

**Comment by golles** [comment]
Thank you for sharing, this would be a really nice improvement

**Comment by hepfister** [comment]
I read that but decided to ask anyway. I understand. Would love to fork it and develop my own. Not a programmer myself but never too old to learn right. Any suggestions where to start code wise would be great.

**Comment by golles** [comment]
In the current implementation, the sensors are defined in const.py https://github.com/golles/ha-kamstrup_403/blob/main/custom_components/kamstrup_403/const.py#L30-L320
And the protocol is defined in https://github.com/golles/ha-kamstrup_403/blob/main/custom_components/kamstrup_403/kamstrup.py

**Comment by TheChatty** [comment]
At the moment [`__init__.py` ](https://github.com/golles/ha-kamstrup_403/blob/main/custom_components/kamstrup_403/__init__.py#L89) loops through all sensors defined in `SENSORS.items()` and calls [`readvar`](https://github.com/golles/ha-kamstrup_403/blob/main/custom_components/kamstrup_403/kamstrup.py#L189) for each one.

The new way would be to pass `SENSORS.items()` down to new function `readvars` which prepares a single request for all sensors. The new request would look like:
```
3F 10 <numSensors> <codeOfSens1> <codeOfSens2> <codeOfSens3> ...
```

And the single response needs to be taken apart matched to each sensor:
```
3F 10 <codeOfSens1> <respSens1> <codeOfSens2> <respSens2> <codeOfSens2> ...
```

See [this example](https://tasmota.github.io/docs/Smart-Meter-Interface/#kamstrup-multical-4xx-6xx-8xx) for how to request 7 sensors at once and how to decipher the response.

**Comment by golles** [comment]
Thanks, my previous comment was supposed to go to #31 (sorry for that)

For now, I marked this as a nice to have in #30. If I won't make it in that PR, I'll do it in the next one

**Comment by golles** [comment]
PR for this feature is available, see #56 

Please, if you have time, test this branch: `29-support-reading-of-multiple-values-at-once`

**Comment by golles** [comment]
FYI: This feature has been released in version `2.1.0`
